css/tabstrip.css

```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
 .tab.selected .tab-thumbnail {
   transform: scale(1);
 }
+
+.tab-close-button {
+  font-family: FontAwesome;
+  text-align: center;
+  font-size: 16px;
+  vertical-align: middle;
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  top: 10px;
+  right: -10px;
+  background: #222;
+  color: white;
+
+  z-index: 2;
+  border-radius: 100%;
+
+  display: none;
+}
+
+
+.tab-close-button:after {
+  content: "\f00d";
+}
+
+.tab:hover .tab-close-button {
+  display: inline-block;
```

Comment:
Nit: `display: block`. This is `position: absolute`, so you can't really use the fancier `display` models.